exercises/practice/roman-numerals/.meta/Example.vb

```diff
@@ -0,0 +1,36 @@
+Imports System.Collections.Generic
+Imports System.Text
+Imports System.Runtime.CompilerServices
+
+Module RomanNumeralExtension
+    Private Shared ReadOnly ArabicToRomanConversions As Dictionary(Of Integer, String) = New Dictionary(Of Integer, String) From {
```

Comment:
```suggestion
    Private ReadOnly ArabicToRomanConversions As New Dictionary(Of Integer, String) From {
```

Shared is only for class-level variable. Also, object creation can be simplified.